frontend/src/App.tsx

```diff
@@ -140,6 +223,27 @@ const NFTForm = () => {
 
 }
 
+/* A component that embeds the console log into the GUI.
```

Comment:
Hm, the intention is good, just this looks way worse than the in-browser developer console . Also exceptions don't show up. 
What about putting some text hint: "Hint: Open developer console to see whats happening"?

Reply:
Oh I never responded to this. I agree with you and I'll remove that component when everything else is done. Leaving it in until then b/c the dev console doesn't use responsive design and is awkward to look at in a small window